components/breadcrumb/index.css

```diff
@@ -18,35 +18,35 @@ governing permissions and limitations under the License.
   --spectrum-breadcrumbs-block-size-multiline: var(--spectrum-breadcrumbs-height-multiline);
 
   /* text regular */
-  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-small);
+  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-100);
   --spectrum-breadcrumbs-font-size: var(--spectrum-font-size-200);
   --spectrum-breadcrumbs-font-family: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
```

Comment:
Since we are using the custom vars token name of **--spectrum-font-family-base**, I think this placeholder note (referencing the original token name provided in the XD) can be removed:

`/* placeholder for --spectrum-font-family-default */`